app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class SessionsController < Devise::SessionsController
+  include Devise::Controllers::Rememberable
+  def create
+    delete_legacy_cookie
+    super
+  end
+
+  def destroy
+    delete_legacy_cookie
+    cookies.delete :remember_user_token, domain: ".#{SiteConfig.app_domain}"
+    # Let's say goodbye to all the cookies when someone signs out.
+    cookies.clear(domain: cookie_domain)
+
+    super
+  end
+
+  private
+
+  def legacy_cookie_domain
+    Rails.env.production? ? SiteConfig.app_domain : nil
+  end
+
+  def cookie_domain
+    ".#{legacy_cookie_domain}"
```

Comment:
`legacy_cookie_domain` can be `nil`, should we guard against this or do we want it to return `"."` in this case?